views/assets/stylesheets/components/buttons.css

```diff
@@ -0,0 +1,37 @@
+.qor-button__icon-add,
+.qor-fieldset__add-promotion,
+button.qor-fieldset__add {
+	font-size: var(--size-xs);
+	font-family: 'OpenSans-Semibold';
+	text-transform: none;
+}
+
+.qor-button__icon-add:after,
+.qor-fieldset__add-promotion:after,
+button.qor-fieldset__add:after {
+	content: "+";
+	font-family: "tykon" !important;
+	font-style: normal !important;
+	font-weight: normal !important;
+	font-variant: normal !important;
+	text-transform: none !important;
+	speak: none;
+	line-height: 1;
+	-webkit-font-smoothing: antialiased;
+	font-size: 10px;
+}
+
+.mdl-button--primary.mdl-button--primary {
```

Comment:
is there a way to replace all buttons, so that none of them would use mdl css classes?

Reply:
same for rest of components

Reply:
The current problem is with the dynamically created elements via JS and Go.
This will be solved for JS after we merge Pato's PR which serves the files from the source and bundles them (currently they are all served via some Go cache). 
For the Go ones, its in my list to find out how they do it.

Reply:
cool! thanks